lib/Backend/IR.h

```diff
@@ -503,13 +504,19 @@ class Instr
 
 class ByteCodeUsesInstr : public Instr
 {
-public:
-    static ByteCodeUsesInstr * New(Func * func);
-    static ByteCodeUsesInstr* New(IR::Instr* originalBytecodeInstr, SymID symid);
+private:
     BVSparse<JitArenaAllocator> * byteCodeUpwardExposedUsed;
-    PropertySym *              propertySymUse;
+    
+public:
+    static ByteCodeUsesInstr * New(IR::Instr * originalBytecodeInstr);
+    static ByteCodeUsesInstr * New(Func * originalBytecodeInstr, uint32 offset);
+    const BVSparse<JitArenaAllocator> * getByteCodeUpwardExposedUsed() const { return byteCodeUpwardExposedUsed; }
 
-    void Set(uint symId);
+    PropertySym *              propertySymUse;
+    void Set(bool isjitoptimizedreg, uint symId);
```

Comment:
I've added two additional set methods; one takes an Opnd* as an argument, and the other just takes a symId (for cases where I was previously just passing "false" statically into the set method). I've gone through and used these where applicable, and added comments such that intellisense/other autocompletion will provide additional information.